src/apps/_experiments/OswAppWeather.cpp

```diff
@@ -288,19 +305,19 @@ bool OswAppWeather::_request() {
         strcpy(encoded_arr, encoded.c_str());
         String encoded_S = String(encoded_arr);
         if (!this->pref.putString("wtr",encoded_S)) {
-            OSW_LOG_E("Error: unable to write to NVS");
+            OSW_LOG_E("Unable to write to NVS?!");
```

Comment:
what is mean last word `?!`

Reply:
Like a surprised / confused (`?`), but shouting (`!`) - I chose confused, because the error-handling is too bad that the message cannot tell why